Description: Removes the useless dependancies
 This was reported by dpkg-shlibs, that warned that our binary was
 marked as depending on various libs without any reason as no symbol
 were used.
Author: Matthew Bekkema <mat8913ftw@gmail.com>
Forwarded: no (we should do it)


---
 src/CMakeLists.txt |    6 ------
 1 file changed, 6 deletions(-)

Index: freeminer-0.4.12.6/src/CMakeLists.txt
===================================================================
--- freeminer-0.4.12.6.orig/src/CMakeLists.txt	2015-02-21 18:58:56.363095752 +0100
+++ freeminer-0.4.12.6/src/CMakeLists.txt	2015-02-21 18:58:56.359095752 +0100
@@ -347,10 +347,6 @@
 	# Unix probably
 	if(BUILD_CLIENT)
 		find_package(X11 REQUIRED)
-		find_package(OpenGL REQUIRED)
-		find_package(JPEG REQUIRED)
-		find_package(BZip2 REQUIRED)
-		find_package(PNG REQUIRED)
 		if(APPLE)
 			FIND_LIBRARY(CARBON_LIB Carbon)
 			FIND_LIBRARY(COCOA_LIB Cocoa)
@@ -372,7 +368,6 @@
 	endif(APPLE)
 	#set(CLIENT_PLATFORM_LIBS -lXxf86vm)
 	# This way Xxf86vm is found on OpenBSD too
-	find_library(XXF86VM_LIBRARY Xxf86vm)
 	mark_as_advanced(XXF86VM_LIBRARY)
 	set(CLIENT_PLATFORM_LIBS ${CLIENT_PLATFORM_LIBS} ${XXF86VM_LIBRARY})
 endif()
